src/libai/nn/NeuralNetwork.java

```diff
@@ -158,7 +158,7 @@ public double error(Matrix[] patterns, Matrix[] answers) {
 	 */
 	public double error(Matrix[] patterns, Matrix[] answers, int offset, int length) {
 		double error = 0.0;
-		Matrix Y = new Matrix(answers[0].getRows(), 1);
```

Comment:
this is just to know the size of the rows in the answer vector. 0 is more likely to not fail than offset.